client.go

```diff
@@ -638,6 +638,13 @@ func (c *Client) Ban(channel, username, reason string) {
 	c.Say(channel, "/ban "+username+" "+reason)
 }
 
+// DeleteMessage delete message in chat
+//
+// Bot user must be an moderator, channel owner message can't be deleted
```

Comment:
Staff can't be deleted either, but I guess that's a more rare usecase, so whatever. We don't need to mention it

Reply:
Nevermind. Staff messages can be deleted